eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -19,24 +20,19 @@ function Sparse-Checkout($branchName, $packagePath)
     {
         Remove-Item $sparseCheckoutFile -Force
     }    
-    git sparse-checkout init --cone
-    git sparse-checkout set $packagePath
+    git sparse-checkout init --no-cone
+    git sparse-checkout set $packagePath $globalPackageJson
     git checkout $branchName
 }
 
 function Generate-Apiview-File($packagePath)
 {
     Write-Host "Generating API review token file from path '$($packagePath)'"
     Push-Location $packagePath
-    try
-    {
-        npm install
-        cadl compile . --emit=@azure-tools/cadl-apiview
-    }
-    finally
-    {
-        Pop-Location
-    }
+    npm install
+    npm list
+    cadl compile . --emit=@azure-tools/cadl-apiview

```

Comment:
Doesn't this also require that you add this package into that global package.json to get it to work?

Reply:
Nevermind it is already added https://github.com/Azure/azure-rest-api-specs/blob/main/package.json#L17